scripts/add-helm-code.sh

```diff
@@ -21,13 +21,26 @@
 # Working directory does not matter.
 
 HOME_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )/.." && pwd )"
+# Directory containing the yaml file
+dir=${HOME_DIR}/chart/templates
 
-if [ "$#" -lt 1 ]; then
-    echo "adds/remove helm tags to cluster scoped files such as ClusterRole and CLusterRoleBinding"
-    echo "Usage: $(basename $0) add | remove"
-    exit
-fi
-CMD=$1
+while (( $# > 0 )); do
+	case "$1" in
+    (-h|--help)
+    	echo "adds/remove helm tags to cluster scoped files such as ClusterRole and CLusterRoleBinding"
+	    echo "Usage: $(basename $0) [--dir directory-containing-yaml] add | remove"
```

Comment:
"containing-yaml" is not very explanatory. A Helm chart has two directories containing YAML: the top-level one, and `templates/`.

Reply:
Agreed.  Changed to chart directory.